shopinvader/models/shopinvader_backend.py

```diff
@@ -109,6 +109,13 @@ class ShopinvaderBackend(models.Model):
         "statistics reasons. A new cart is created automatically when the "
         "customer will add a new item.",
     )
+    validate_customers = fields.Boolean(
+        default=False,  # let's be explicit here :)
+        help="Turn on this flag to block non validated customers. "
+        "If customers' partner are not validated, "
+        "registered users cannot log in. "
+        "Salesman will get notified via mail activity.",
```

Comment:
@simahawk We should put into the help text that this validation process only occurs for customers who are a company....

Reply:
I'd like to extend this to all kind of users and maybe limit to company only via another flag.

Reply:
That would be great!